disassemblers/ofrak_angr/ofrak_angr_test/test_unpackers.py

```diff
@@ -54,14 +54,30 @@ async def expected_results(self, unpack_verify_test_case: ComplexBlockUnpackerTe
                 0x110,
                 0x110,
                 0x130,
-                keep_same_is_exit_point=True,
+                keep_same_is_exit_point=False,
             )
 
             return self._fixup_test_case_for_pie(
                 unpack_verify_test_case.expected_results,
                 pie_base_vaddr=0x400000,
             )
 
+        elif unpack_verify_test_case.binary_md5_digest == "c79d1bea0398d7a9d0faa1ba68786f5e":
+            # Latest version of angr misses this DataWord now = the ref to it does not appear in the list of xrefs
```

Comment:
Can we specify which version specifically in this comment?

Reply:
Done.